packages/components/table/examples/DynamicCreation.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Table } from '@contentful/f36-components';
+import { Badge } from '@contentful/f36-components';
+
+export default function DynamicCreation() {
```

Comment:
a small optional thing: it would be nice to have `Table` in the file and component name. It simplifies search and also is consistent with other examples.